guides/introduction/overview.md

```diff
@@ -6,11 +6,37 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, susci
+Depending on where you are at in your programming journey here are some additional tools you may want to consider installing.
+
+### Text Editors
+
+A text editor is at the core of a developer's toolbox, so we shoud choose it carefully.
+
+#### Visual Studio Code (Recomended)
+
+[Installation instruction.](https://code.visualstudio.com/docs/setup/setup-overview)
+
+Recommended extensions for editor ease to develoepment workflow.
+
+- [ElixirLS](https://marketplace.visualstudio.com/items?itemName=JakeBecker.elixir-ls) - Elixir support with debugger, autocomplete, and more.
+- [LiveShare](https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare) - Real-time collaborative development from the comfort of your favorite tools. Could help in pair programming with mentor during workshops.
+- [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) - This is an opinionated code formatter. It enforces a consistent style by parsing your code and re-printing it with its own rules that take the maximum line length into account, wrapping code when necessary.
+- [Indenticator](https://marketplace.visualstudio.com/items?itemName=SirTori.indenticator) - Visually highlights the current indent depth.
+
+#### Alternatives
+
+- [GitHub Atom](https://atom.io)
+- [Sublime](https://www.sublimetext.com)
+- [Textmate](https://macromates.com)
+- [Emacs](https://www.gnu.org/software/emacs/)
+- [Vim](https://www.vim.org) (not recommended for a begginers)
+
+
+### Git tools (comming soon)
 
 ## Try Elixir
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, suscipit a, scelerisque sed, lacinia in, mi. Cras vel lorem. Etiam pellentesque aliquet tellus. Phasellus pharetra nulla ac diam. Quisque semper justo at risus. Donec venenatis, turpis vel hendrerit interdum, dui ligula ultricies purus, sed posuere libero dui id orci. Nam congue, pede vitae dapibus aliquet, elit magna vulputate arcu, vel tempus metus leo non est. Etiam sit amet lectus quis est congue mollis. Phasellus congue lacus eget neque. Phasellus ornare, ante vitae consectetuer consequat, purus sapien ultricies dolor, et mollis pede metus eget nisi. Praesent sodales velit quis augue. Cras suscipit, urna at aliquam rhoncus, urna quam viverra nisi, in interdum massa nibh nec erat.
+[Try Elixir course](http://try-elixir.herokuapp.com). This is meant to be an introduction to Elixir, to get familiar with basic concepts. Data is deleted after 5 minutes of idle and many Elixir modules are restricted for security reasons.
```

Comment:
This is meant as an introduction to Elixir by getting familiar with basic concepts. Data is deleted after 5 minutes of being idle and many Elixir modules are restricted due to security reasons.